src/app/ngrx/actions/chart-toggle.action.creator.ts

```diff
@@ -0,0 +1,34 @@
+import { Action } from '@ngrx/store';
```

Comment:
These are some of the few synchronous actions in the app. This is a change in functionality from keeping this info about what episodes and if the podcast is charted on the route. A page of episode guids is too long to put on the route and it just didn't seem useful to carry over. So now it's just being handled in the app state.